hooks/useLocalStorage.ts

```diff
@@ -1,5 +1,7 @@
 
-const useLocalStorage = ((key: string) => {
+type Key = 'cart' | 'favorites'
```

Comment:
Jak  już będzie ten obiekt z kluczami, to można z niego wziąć ten typ i mieć jedno źródło prawdy
```typescript
type LocalStorageKeys = keyof typeof LOCAL_STORAGE_KEYS
```